db/R/dbfiles.R

```diff
@@ -253,7 +254,7 @@ dbfile.posterior.insert <- function(filename, pft, mimetype, formatname, con, ho
 ##'
 ##' This will check the dbfiles, inputs, machines and formats tables to see if the
 ##' file exists
-##' @name dbfile.check
+##' @name dbfile.posterior.check
```

Comment:
The @name on Line 257 Goes with function on line 270 which is dbfile.posterior.check. There was a mismatch earlier that stuck ```dbfile.posterior.check``` and ```dbfile.input.insert``` along with ```dbfile.check``` in the same .Rd file. This changes it so that each function has their own .Rd file. If this is how it's supposed to be I can change it back.